recipes/magnum/all/test_package/test_package.cpp

```diff
@@ -0,0 +1,20 @@
+#include <iostream>
+#include "Magnum/Math/Vector.h"
+#include "Magnum/Math/StrictWeakOrdering.h"
+
+/*
+    I would like to use some windowless application to test, like
+    https://github.com/mosra/magnum-bootstrap/tree/windowless
+    but it doesn't work in CI.
```

Comment:
Some more information would be useful.
e.g. that `EGL_NOT_INITIALIZED` is returned on CI
https://github.com/conan-io/conan-center-index/pull/7095#issuecomment-913841567